### DISCUSSION TITLE
Welcome to doc2pen Discussions!

### DISCUSSION THREAD

**Discussion by smaranjitghose** [smaranjitghose/doc2pen#311] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ashuvssut** [comment]
We Should use functional component instead of class based components.

functional State Management with React Hooks are 'easy to understand' because they are short and simple.

If you like this idea then I will put these changes in #407

**Reply by smaranjitghose** [reply]
Looking forward to it

**Comment by ashuvssut** [comment]
Sir,  we are using bootstrap. I read its implementations in this project. I think it is used to make responsive components in this project.

Sir, is it used for responsiveness only or is there any other purpose too?

**Comment by ashuvssut** [comment]
Sir, lets use material-ui components in our react app. it has beautiful components. we dont have to use bootstrap coz Material ui components have breakpoints which adds to responsive components.

If you like this idea then I will use material-ui components starting from #410 and so on.

**Comment by leesha19** [comment]
hey i know some html and css can i help or is there anything i can do ??

**Reply by anushbhatia** [reply]
Hey @leesha19

You can look through the website and if you find any issue relevant or any bugs that you many encounter or a feature request you can create an issue and we will assign that to you.

**Reply by leesha19** [reply]
ok i have mentioned the issue ....actually when i zoomed the page all the text was moved down  since the color was white it was not visible to see and the checkout button was also dislocated
